src/Hw.tsx

```diff
@@ -0,0 +1,30 @@
+import { useState } from "react";
+import "./Hw.css";
+
+function Hw() {
+    const [state, setState] = useState([0, 1, 2]);
```

Comment:
```suggestion
    const [state, setState] = useState<number[]>([0, 1, 2]);
```

물론 초기값이 있을 때는 제너릭으로 타입을 지정해주지 않아도 되지만,  (타입 추론이 되기 때문에)

하지만 가독성이나 없는 경우도 있으니 저는 가급적이면 쓰는 방향을 추천드려요!!